File: Aulas java eclipse/aula_07/src/aula_07/MediaTurma.java
package aula_07;

import java.util.Scanner;

public class MediaTurma {
//Esse programa é para calcular a média aritimetica das notas de uma turma
	
public static void main(String[] args) {
Scanner leitor = new Scanner(System.in);

//Variáveis
	double nota = 0;
	double soma = 0;
	double quantidade = 0;
	double media = 0;
	
//Entrada de dados
	do {
		System.out.print("Digite a nota da prova: ");
			nota = leitor.nextDouble();
		soma += nota;
		quantidade++;
		
		System.out.print("Digite <1> se houver mais alunos e <0> se não houver: ");
	} while (leitor.nextInt() == 1);
	
	leitor.close();
	
//Processamento
	media = (soma / quantidade);
	
//Saida de dados
	System.out.println("Media da turma: " + media);
}
}

File: Python/Aula02/MediaTurma.py
#Esse programa é para calcular a média aritmetica de uma turma

count = 0
soma = 0
media = 0

while True:
    numero = int(input("Digite a nota: "))
    if numero == -1:
        break
    count += 1
    soma = soma + numero
    media = soma / count

print("O número de alunos é: ", count)
print("A média da turma é: ", media)
